index.html

```diff
@@ -59,17 +59,17 @@
 							</a>
 					</div>
 					<a href="https://github.com/fossasia/labyrinth">
-						<img class="brand-logo center" alt="FOSSASIA" src="labyrinthlogofull.png" width="80" height="60">
```

Comment:
Why removed the logo ? Any reasons ?

Reply:
See this : https://github.com/fossasia/labyrinth/pull/560#issuecomment-357964941